### PR TITLE
ETL-3544: Jobstate to json tool

### PR DESCRIPTION
**Pull request by ydailinkedin**
The JobStateToJsonConverter tool is fixed by @sahilTakiar and checked in branch "statestore-checker". I tested it and added one more option to output to a file.
@sahilTakiar Can you review?

### CODE REVIEW

File: gobblin-runtime/src/main/java/gobblin/runtime/util/JobStateToJsonConverter.java [hunk 1]
```diff
@@ -210,6 +219,12 @@ public static void main(String[] args)
       }
     }
 
-    System.out.println(stringWriter.toString());
+    if (cmd.hasOption('t')) {
+      FileWriter fw = new FileWriter(cmd.getOptionValue('t'));
+      fw.write(stringWriter.toString());
+      fw.close();
```
Comment (liyinan926) [inline gobblin-runtime/src/main/java/gobblin/runtime/util/JobStateToJsonConverter.java]: Please use a `Closer`.

File: gobblin-runtime/src/main/java/gobblin/runtime/util/JobStateToJsonConverter.java [hunk 2]
```diff
@@ -180,6 +182,12 @@ public static void main(String[] args)
         .withDescription("Whether to keep all configuration properties")
         .withLongOpt("keepConfig")
         .create("kc");
+    Option outputToFile = OptionBuilder
+        .withArgName("Output file name")
+        .withDescription("Whether to write the output to a file")
```
Comment (liyinan926) [inline gobblin-runtime/src/main/java/gobblin/runtime/util/JobStateToJsonConverter.java]: The description does not match well with the `argName`.

**Comment by sahilTakiar** [comment]
LGTM

**Comment by chavdar** [comment]
Please create a github wiki for the tool.

**Pull request merged**